packages/contracts-bedrock/snapshots/abi/L2ToL2CrossDomainMessenger.json

```diff
@@ -0,0 +1,226 @@
+[
+  {
+    "inputs": [],
+    "name": "MESSAGE_VERSION",
```

Comment:
hmm this looks like `MESSAGE_VERSION` is still part of the abi, it should be `messageVersion`

Reply:
update for this on the way!